lib/cog/adapters/hipchat/api.ex

```diff
@@ -14,7 +14,7 @@ defmodule Cog.Adapters.HipChat.API do
   end
 
   def lookup_room("@" <> handle) do
-    {:ok, user} = lookup_user(name: "@" <> handle)
+    {:ok, user} = lookup_user(handle: handle)
```

Comment:
In Elixir names and function signatures are combined to convey intent. For instance `lookup_user` is used for both looking up by handle and by id. In a language that doesn't use pattern matching I would expect there to be `lookup_user_by_handle` and `lookup_user_by_id` functions. So, in my mind, `lookup_user(handle: handle)` and `lookup_user(id: id)` feel like two independent functions. Where as something like `is_true(true)` and `is_true(false)` feel like two pieces of a single function and would be grouped together without newlines separating them.

I'm ok with whatever we decide as a group though. I'm not even sure if how I interpreted Elixir's style as the reason it was written that way or not.